src/adapter/src/catalog/builtin.rs

```diff
@@ -1627,7 +1627,8 @@ s.name AS nspname,
 NULL::pg_catalog.oid AS nspowner,
 NULL::pg_catalog.text[] AS nspacl
 FROM mz_catalog.mz_schemas s
-JOIN mz_catalog.mz_databases d ON (d.id IS NULL OR d.name = pg_catalog.current_database())",
+LEFT JOIN mz_catalog.mz_databases d ON (d.id = s.database_id)
```

Comment:
The parens were added to disambiguate some stuff that's now gone, and now they don't match the `ON` style of some other joins. They should be consistent, and so probably removed because there's only one clause.

```suggestion
LEFT JOIN mz_catalog.mz_databases d ON d.id = s.database_id
```